src/arqma_mq/INotifier.h

```diff
@@ -26,52 +25,26 @@
 // INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
 // STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF
 // THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+//
+// Parts of this file are copyright (c) 2014-2020 The Arqma Network
+// Parts of this file are copyright (c) 2020-2021 Gary Rusher
```

Comment:
are we in the future already??
That line is not needed cus Copyright are already stated on top of the file.